StripePaymentSheet/StripePaymentSheetTests/PaymentSheet/STPElementsSessionTest.swift

```diff
@@ -262,6 +262,41 @@ class STPElementsSessionTest: XCTestCase {
 
         XCTAssertEqual(.legacy, savePaymentMethodConsentBehavior)
     }
+    func testSetAsDefault_enabled() {
```

Comment:
Can we update testDecodedObjectFromAPIResponseMapping to make sure that paymentMethodSetAsDefaultForPaymentSheet defaults to false?